readme.md

```diff
@@ -92,7 +101,7 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 
 * **Question:** When does `app.use()` get called?
 
-* **Your Answer:**
+* **Your Answer:** Everytime a request is made
```

Comment:
Remember in general app.use() only fires when a request reaches that line in the route chain.